scripts/extensions/sams/src/store/sets/actions.ts

```diff
@@ -83,18 +75,22 @@ export function loadSets(): IThunkAction<Array<ISetItem>> {
 }
 
 export function openDeleteConfirmationModal(set: ISetItem): IThunkAction<boolean> {
-    return (dispatch, _getState, {superdesk}) => {
+    return (_dispatch, _getState, {superdesk}) => {
         const {gettext} = superdesk.localization;
         const {confirm} = superdesk.ui;
 
-        dispatch(setDeleteConfirmationModalOpen(true));
+        const el = document.createElement('div');
+
+        el.classList.add('modal__backdrop', 'fade', 'in');
```

Comment:
The comments outline an issue between ui-framework and pure React modals. To fix this I would have to implement a custom React component just for a confirmation dialog. So for now this will do, but the underlying problem will need to be fixed in general